### PR TITLE
hover color; attributes; tooltip text

### PR DESCRIPTION
**Pull request by anthonybaldwin**
* change `fill:` to `currentColor` as this is what Discord uses and will take care of hover state
* added attributes I found when attempting to add a hover state myself before I found `currentColor`; I left them 🤷‍♂️
* changed tooltip text to `Show` and `Hide` when using toolbar button; imo, this better matches the other buttons

### CODE REVIEW

**Comment by anthonybaldwin** [comment]
Maybe audio next 🤞

File: index.js [hunk 1]
```diff
@@ -89,12 +89,15 @@ module.exports = class GameActivityToggle extends Plugin {
 
                 r.props.children.unshift(React.createElement(r.props.children[0].type, {
                     icon: () => showCurrentGame ? enabledIcon('20') : disabledIcon('20'),
+                    'aria-label': `${showCurrentGame ? 'Hide' : 'Show'} Game Activity`,
+                    'aria-checked': `${showCurrentGame ? 'false' : 'true'}`,
+                    role: "switch",
```
Comment (Juby210) [inline index.js]: useless (also `aria-label` is added by tooltipText, so you don't need to add it manually)
```suggestion
```

File: index.js [hunk 2]
```diff
@@ -89,12 +89,15 @@ module.exports = class GameActivityToggle extends Plugin {
...
                     onClick: () => {
                         showCurrentGame = !showCurrentGame
                         settings.updateRemoteSettings({ showCurrentGame })
                         forceUpdateElement('.' + classes.container, true)
                     },
-                    tooltipText: Messages.SHOW_CURRENT_GAME
+                    tooltipText: `${showCurrentGame ? 'Hide' : 'Show'}`
```
Comment (Juby210) [inline index.js]: `tooltipText` is used for tooltip text, not `aria-label`, so it should show what you want to hide or show
```suggestion
                    tooltipText: `${showCurrentGame ? 'Hide' : 'Show'} Game Activity`
```

**Review by Juby210 — CHANGES_REQUESTED** [review]

File: index.js (hunk 1 above)
Comment (anthonybaldwin) [reply index.js]: Neither of the attributes are added to the button AFAICT, but they do not seem to affect the functionality of this particular button. Tidier without for sure.

**Review by Juby210 — APPROVED** [review]

File: index.js (hunk 1 above)
Comment (Juby210) [reply index.js]: image

**Pull request merged**